packages/core/strapi/lib/core-api/controller/collection-type.js

```diff
@@ -31,8 +31,10 @@ const createCollectionTypeController = ({ contentType }) => {
      * @return {Object}
      */
     async findOne(ctx) {
+      const sanitizedQuery = await this.sanitizeQuery(ctx);
```

Comment:
```suggestion
      const { query } = await this.sanitizeQuery(ctx);
```
We can do something like this everywhere now since sanitizedQuery isn't used anywhere except for this purpose.

Reply:
sanitizeQuery returns the query as a whole, you're not supposed to destructure it or extract something from it :thinking: (except populate, fields, etc...).